101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "The region where resources are deployed"
+      },
+      "defaultValue": "[resourceGroup().location]"
+    },
+    "logicAppName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Logic App"
+      }
+    },
+    "eventGridTopicName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Event Grid Topic"
+      }
+    },
+    "eventGridTopicSchema": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Topic schema"
+      },
+      "allowedValues": [
+        "EventGridSchema",
+        "CloudEventV01Schema"
+      ],
+      "defaultValue": "CloudEventV01Schema"
+    },
+    "eventGridSubscriptionName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Event Grid Subscription"
+      }
+    },
+    "eventGridSubscriptionDeliverySchema": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription delivery schema"
+      },
+      "allowedValues": [
+        "EventGridSchema",
+        "CloudEventV01Schema"
+      ],
+      "defaultValue": "CloudEventV01Schema"
+    },
+    "eventGridSubscriptionEndpointType": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription endpoint type"
+      },
+      "allowedValues": [
+        "Webhook"
+      ],
+      "defaultValue": "Webhook"
+    },
+    "eventGridSubscriptionIncludedEventTypes": {
+      "type": "string",
+      "metadata": {
+        "description": "Comma delimited list of filters for Event Grid Subscription"
+      },
+      "defaultValue": "All"
+    },
+    "eventGridSubscriptionIsSubjectCaseSensitive": {
+      "type": "bool",
+      "metadata": {
+        "description": "Value indicating whether the filter subjectBeginsWith and subjectEndsWith are case sensitive or not"
+      },
+      "defaultValue": false
+    }
+  },
+  "variables": {
+    "logicApp": {
+      "name": "[parameters('logicAppName')]",
+      "apiVersion": "2017-07-01",
```

Comment:
remove this

Reply:
As that `apiVersion` is re-used within the ARM template a few times, I intentionally extract that value to the `variables` section. Is it an issue?

Reply:
Removed